specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/GA/2021-02-01/examples/GetClustersByResourceGroupExample.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
+    "resourceGroupName": "k8sc-rg",
+    "api-version": "2021-02-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "72f988bf-86f1-41af-91ab-2d7cd011db48",
```

Comment:
I think the id field here in the examples are wrong. They should be ARM ids.

Reply:
updated old examples as well